rcl_lifecycle/include/rcl_lifecycle/transition_map.h

```diff
@@ -99,18 +101,19 @@ rcl_lifecycle_transition_map_fini(
  * Uses Atomics       | No
  * Lock-Free          | Yes
  *
- * \param[in] map to be modified
+ * \param[in] transition_map to be modified
  * \param[in] state the state to register
  * \param[in] allocator a valid allocator used to register the state machine
  * \return `RCL_RET_OK` if the state was registered successfully, or
  * \return `RCL_RET_INVALID_ARGUMENT` if any arguments are invalid, or
+ * \return `RCL_RET_LIFECYCLE_STATE_REGISTERED` if state is already registered, or
  * \return `RCL_RET_ERROR` if an unspecified error occurs.
```

Comment:
I think this function can not return RCL_RET_ERROR anymore, right?

Reply:
Yes.
I will update the doc.